### ISSUE TITLE
The plugin is not working with import "sap/ui/test/gherkin/opa5TestHarness"

### ISSUE THREAD

**Issue by JuanjoPP** [SAP/karma-openui5#14]
We have an `allJourneys.js` script that we want to be able to test using Karma-Openui5. The situation is that, when this script has this content, it runs without problems (executing no tests):

```
sap.ui.require(
  [
    "jquery.sap.global",
    "sap/ui/test/Opa5",
    "test/integration/arrangements/Common",
    "test/integration/pageObjects/MainPage",
    "test/integration/steps/SkeletonSteps"
  ],
  function($, Opa5, Common, SkeletonSteps, opa5TestHarness) {
    "use strict";
    Opa5.extendConfig({
      viewNamespace: "com.xpr.customer.skeleton.view.",
      arrangements: new Common()
    });

    /*opa5TestHarness.test({
      featurePath: "test.integration.Skeleton",
      steps: SkeletonSteps,
      generateMissingSteps: true
    });*/
  }
);
```

However, when we add an import to sap/ui/test/gherkin/opa5TestHarness, like so:
```
sap.ui.require(
  [
    "jquery.sap.global",
    "sap/ui/test/Opa5",
    "test/integration/arrangements/Common",
    "test/integration/pageObjects/MainPage",
    "test/integration/steps/SkeletonSteps",
    "sap/ui/test/gherkin/opa5TestHarness"
  ],
  function($, Opa5, Common, SkeletonSteps, opa5TestHarness) {
    "use strict";
    Opa5.extendConfig({
      viewNamespace: "com.xpr.customer.skeleton.view.",
      arrangements: new Common()
    });

    /*opa5TestHarness.test({
      featurePath: "test.integration.Skeleton",
      steps: SkeletonSteps,
      generateMissingSteps: true
    });*/
  }
);
```

It fails, leaving this trace:
```
Firefox 60.0.0 (Windows 10 0.0.0) ERROR
  {
    "message": "TypeError: q is null\nat https://openui5.hana.ondemand.com/resources/sap-ui-core.js line 104 > eval:6:1282\n\n@https://openui5.hana.ondemand.c
om/resources/sap/ui/qunit/qunit-junit.js?eval:6:1282\nrunLoggingCallbacks@C:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qu
nit/qunit/qunit.js:1047:5\nbegin@C:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/qunit/qunit.js:3128:5\nscheduleBegin/
<@C:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/qunit/qunit.js:3095:6\nW@https://openui5.hana.ondemand.com/resources
/sap-ui-core.js:327:824\n",
    "str": "TypeError: q is null\nat https://openui5.hana.ondemand.com/resources/sap-ui-core.js line 104 > eval:6:1282\n\n@https://openui5.hana.ondemand.com/r
esources/sap/ui/qunit/qunit-junit.js?eval:6:1282\nrunLoggingCallbacks@C:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/
qunit/qunit.js:1047:5\nbegin@C:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/qunit/qunit.js:3128:5\nscheduleBegin/<@C:
/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/qunit/qunit.js:3095:6\nW@https://openui5.hana.ondemand.com/resources/sap
-ui-core.js:327:824\n"
  }
```

Running debug mode (using the button in the navigator that Karma launches) in this state leaves this trace:
```
TypeError: q is null[Learn More]
sap-ui-core.js%20line%20104%20%3E%20eval:6:1282
<anonymous>
https://openui5.hana.ondemand.com/resources/sap/ui/qunit/qunit-junit.js:6:1282
runLoggingCallbacks
http://localhost:9876/absoluteC:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/qunit/qunit.js:1047:5
begin
http://localhost:9876/absoluteC:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/qunit/qunit.js:3128:5
scheduleBegin/<
http://localhost:9876/absoluteC:/Users/Juan%20Jose%20Perez/Documents/xerox-ui/xpr-walking-skeleton/node_modules/qunit/qunit/qunit.js:3095:6
W
XHRProxy-dbg.js:98:15
```
It seems opa5TestHarness calls a script called qunit-junit.js, that doesn't play nice with Karma.

**Comment by JuanjoPP** [comment]
After some more testing apparently in OpenUI5 version 1.52 it works well, but in 1.54 fails.

**Comment by matz3** [comment]
Seems like https://github.com/SAP/openui5/commit/cf4228633b559d0b4f39721e2bf78d00b7c0e620 broke this. A fix will be pushed soon. Thanks for reporting 👍 

A workaround would be to enable the QUnit UI like in our sample app:
https://github.com/SAP/openui5-sample-app/blob/eaadd191810f9f1b2672bf582d0e253c6cc4898f/Gruntfile.js#L155-L160

**Comment by matz3** [comment]
Should now be fixed in 1.54, 1.56 and master branch.
